### PR TITLE
feat: create ticket details sidebar

### PR DESCRIPTION
**Pull request by nitzanh2000**

### CODE REVIEW

File: src/routes/current-ticket/CurrentTicketPage.jsx [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+      link: ''
+    }
+  ]
+};
+
 export const CurrentTicketPage = () => {
+  const [ticket, setTicket] = useState(defaultTicket);
+
+  const addTeam = newTeam => {
+    setTicket(x => {
```
Comment (nitzanh2000) [inline src/routes/current-ticket/CurrentTicketPage.jsx]: name

File: src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import InsertDriveFileIcon from '@mui/icons-material/InsertDriveFile';
+import UploadFileIcon from '@mui/icons-material/UploadFile';
+import { Box, Paper, Typography } from '@mui/material';
+import { useState } from 'react';
+;
+
+export const TicketFileList = ({files}) => (
+  <Box display="flex" flexWrap="wrap" gap={2}>
+    <UploadFileButton onClick={console.log}/>
```
Comment (nitzanh2000) [inline src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: log

File: src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+);
+
+
+const File = ({title}) => {
+  const [isHovering, setIsHovering] = useState(false);
+  
+  return (
+    <Paper
+      sx={{display: 'flex', flexDirection: 'column', alignItems: 'center', p: 2, cursor: 'pointer'}}
+      elevation={isHovering ? 3 : undefined}
```
Comment (nitzanh2000) [inline src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: null

File: src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  const [isHovering, setIsHovering] = useState(false);
+
+  return (
+    <Paper
+      sx={{
+        display: 'flex', flexDirection: 'column', alignItems: 'center', p: 2, cursor: 'pointer',
+        color: theme => theme.palette.primary.light
+      }}
+      onClick={onClick}
+      elevation={isHovering ? 3 : undefined}
```
Comment (nitzanh2000) [inline src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: null

Comment (dorilahav) [reply src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: default value for elevation only triggers when undefined

File: src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx (hunk 3 above)
Comment (dorilahav) [reply src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: default value for elevation only triggers when undefined

File: src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx (hunk 2 above)
Comment (dorilahav) [reply src/routes/current-ticket/ticket-details-sidebar/TicketFileList.jsx]: k

**Pull request merged**